beacon_node/network/src/sync/range_sync/chain.rs

```diff
@@ -1087,6 +1107,36 @@ impl<T: BeaconChainTypes> SyncingChain<T> {
         Ok(KeepChain)
     }
 
+    /// Checks all custody column subnets for peers. Returns `true` if there is at least one peer in
+    /// every custody column subnet.
+    fn good_peers_on_custody_subnets(&self, epoch: Epoch, network: &SyncNetworkContext<T>) -> bool {
+        if network.chain.spec.is_peer_das_enabled_for_epoch(epoch) {
```

Comment:
We can probably lift the `if` condition up to the caller side, so it's not too misleading here

Reply:
hmm realised  this is called in mutiple places, might be better to leave it here.